packages/node_modules/@webex/react-component-space-item/src/index.js

```diff
@@ -100,13 +97,12 @@ function SpaceItem({
     hasCallSupport && callStartTime &&
     <Button
       ariaLabel="Join Call"
-      callStartTime={callStartTime}
       color="green"
       size={28}
       onClick={handleCallClick}
     >
       {
-      hasJoinedOnThisDevice
+      callStartTime
```

Comment:
No longer matters if you've joined the call or not, it will always display the call progress timer.